tests/test_live.py

```diff
@@ -90,3 +91,34 @@ async def test_gemini_structured(http_client: httpx.AsyncClient):
     print('Gemini structured cost:', result.cost())
     cost = result.cost()
     assert cost.total_tokens is not None and cost.total_tokens > 0
+
+
+async def test_groq(http_client: httpx.AsyncClient):
```

Comment:
I guess I'll need to set an API key for this?

I can look it up, but is the process trivial?

Reply:
Free API key can be created, trivial just go to https://console.groq.com/

Reply:
I'll fix this once the initial PR is merged, `test-live` will no longer run on PRs from forks once you rebase.